City-of-Helsinki/drupal-helfi-platform-config
# UHF-3206 Shareable image token

How to test with running instance:
- Make sure your instance is up and running
- Update the modules/themes
  - `composer require drupal/hdbt:dev-UHF-3206_shareable_image_token && composer require drupal/helfi_platform_config:dev-UHF-3206_shareable_image_token && composer require drupal/helfi_proxy:dev-UHF-3206`
- Run updates: `make drush-cim drush-updb drush-cr`
- Log in if you haven't already `make drush-uli`
- Go to front page and view the page source
   - Make sure you can find og:image and twitter:image from the `<head />` tag
   `<meta property="og:image" content="//helfi-something.docker.so/...">
     <meta name="twitter:image" content="//helfi-something.docker.so/...">` 
   - If there is no liftup image, meaning the meta tags above points to `.../src/images/og-global.png`, edit the page and add a liftup image (can be selected from the sidebar)
   - After saving the node, make sure the meta tags above points to the selected image
- Go to any Basic page (considering that front page is a landing page) and repeat the steps above
- Go to any Unit and check that the meta properties exist
- Notice! If the meta tags do not exist in source code, check the node for overridden settings. These can be found from the sidebar under "Meta information". The correct token for nodes is `[node:shareable-image]`

Also check these PRs:
https://github.com/City-of-Helsinki/drupal-hdbt/pull/194
https://github.com/City-of-Helsinki/drupal-module-helfi-proxy/pull/19/files
